progress/views/index.html

```diff
@@ -0,0 +1,8 @@
+<index: attributes="accent class">
+  <div class="dmd-progress-container{{if loading}} dmd-progress-container--showing{{/if}}">
```

Comment:
By the way, the class attribute should always be on the top most element in order for it to make sense. Now, if one wants to do something about the container - e.g. by adding a custom class - that's not possible.